assets/questions/reverseWords.js

```diff
@@ -395,10 +427,33 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
       ]
     }, {
       buggyFunctionName: 'AuxiliaryCode.useWeakInequalitiesForCharValidity',
+      ignoredTestSuiteIds: [],
       messages: [
         'Note that numbers should not be treated as English words.'
       ]
     }],
+    suiteLevelTests: [{
+      testSuiteIdsThatMustPass: ['SAMPLE_INPUT', 'GENERAL_CASE', 'WHITESPACE'],
+      testSuiteIdsThatMustFail: ['CORNER_CASES'],
+      messages: [
+        [
+          'Are you happy with the code? It must give the correct answer in ',
```

Comment:
I don't think the first sentence is appropriate. Maybe "It seems like your code is failing on some test cases. Try stepping through some sample inputs on paper and see if your output is consistent with your expected output in all cases."

Reply:
Done. Thank you for the catch!